### PR TITLE
Feature/enable ci

### PR DESCRIPTION
**Pull request by jordenk**
* Skip broken tests.
* Enable CI

### CODE REVIEW

File: .github/workflows/run_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: run-tests
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (jordenk) [inline .github/workflows/run_tests.yaml:6]: I'm not too familiar with Firebase, but it looks like we would run the app on linux.

File: .github/workflows/run_tests.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x]
```
Comment (jordenk) [inline .github/workflows/run_tests.yaml:9]: I didn't see a node version specified in the project, so I'm assuming that we're just using node as a script runner. I just picked the latest versions.

File: .github/workflows/run_tests.yaml (hunk 1 above)
Comment (daningenthron) [reply .github/workflows/run_tests.yaml:6]: Same strategy for the backend works here.

**Review by daningenthron — APPROVED** [review]
Looks great thanks Jorden!

**Pull request merged**